SmartDeviceLinkTests/Notifications/SDLResponseDispatcherSpec.m

```diff
@@ -714,11 +714,6 @@
                 [testDispatcher storeRequest:testPerformAudioPassThru handler:nil];
             });
             
-            it(@"should store the handler" ,^{
-//                expect(testDispatcher.audioPassThruHandler).toNot(beNil());
-//                expect(testDispatcher.audioPassThruHandler).to(equal(testPerformAudioPassThru.audioDataHandler));
```

Comment:
You can get these to compile by casting `audioPassThruHandler` to `id`
```
expect((id)testDispatcher.audioPassThruHandler).to(equal((id)testPerformAudioPassThru.audioDataHandler));
```

Reply:
Blocks aren’t an `id` though so I suspect that casting them to that isn’t doing a proper comparison.